webapp/webpack.config.babel.js

```diff
@@ -52,13 +53,28 @@ let config = {
       },
       {
         test: /\.css$/,
-        exclude: /node_modules/,
-        loaders: ['style', 'css?modules&sourceMap&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]', 'postcss']
+        include: path.resolve(__dirname, "src"),  // limit match to only src/
+        exclude: path.resolve(__dirname, "src/public.css"),
+        loader: ExtractTextPlugin.extract(
+          'style',
+          'css?modules&sourceMap&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]',
+          'postcss'
+        )
+      },
+      {
+        test: path.resolve(__dirname, "src/public.css"),
```

Comment:
right @ragingsquirrel3 ! Actually, I was gonna ask you about this :) There's slight distinction between what's in `public/` directory and this `public.css`. I think stuff in `public/` doesn't need to get a content based hash, because they won't change. But this `public.css` seems to be a collection of overriding external stylesheets (such as flybase-data-grid), so it probably need a hash. Would be nice to hear your thoughts on 1) how should the`public/` directory be used 2) how to use and override external stylesheets